server/api/crud/model_monitoring/deployment.py

```diff
@@ -422,10 +341,7 @@ def deploy_model_monitoring_writer_application(
 
     def _initial_model_monitoring_stream_processing_function(
         self,
-        project: str,
-        model_monitoring_access_key: str,
         stream_image: str,
-        auth_info: mlrun.common.schemas.AuthInfo,
```

Comment:
Need to remove them from docstring as well.

Reply:
f3e87a4